webapp/publisher/snaps/views.py

```diff
@@ -351,9 +355,9 @@ def post_register_name():
             registrant_comment=registrant_comment,
         )
     except StoreApiResponseErrorList as api_response_error_list:
-        user = publisher_api.get_account(flask.session)
+        stores = admin_api.get_stores(flask.session)
```

Comment:
The account no longer has the list of stores, so get the list of stores directly from the API